traits/observation/tests/test_expression.py

```diff
@@ -683,6 +683,22 @@ def test_equality_different_type(self):
         self.assertNotEqual(expr, "1")
 
 
+class TestObserverExpressionSlots(unittest.TestCase):
+    """ Check that expressions use __slots__. """
+
+    def test_single_expression(self):
+        expr = create_expression(1)
+        self.assertFalse(hasattr(expr, "__dict__"))
```

Comment:
nitpicky question - why not use the same tests as earlier where you check for an exception instead of a different test?

Reply:
Meh. That was this morning; this afternoon, I like this way better. (But not enough better to go back and update all the other tests.)